src/bsk_rl/envs/general_satellite_tasking/scenario/sat_actions.py

```diff
@@ -126,7 +129,7 @@ def __init__(
             super().__init__(*args, **kwargs)
             setattr(self, fsw_action + "_duration", action_duration)
 
-            def act(self, prev_action_key=None) -> str:
+            def act(self, prev_action_key=None, always_reset=always_reset) -> str:
```

Comment:
Does `always_reset` need to be an argument here?

Reply:
It doesn't. I agree that it is better to remove it from there.